code/src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import moment from 'moment'
+import { todos } from "reducers/todos";
+import { useSelector, useDispatch } from "react-redux";
+
+export const Header = () => {
+  const list = useSelector((store) => store.todos.list);
+  const doneList = list.items.filter((item) => item.done);
+  const dispatch = useDispatch();
+  
+  const onRemoveClicked = (event) => {
+    dispatch(todos.actions.removeAllTodos());
+  };
+
+  return (
+    <div className="header-container">
```

Comment:
Just a minor thing but remember to still use semantic html elements, so maybe not divs everywhere but use sections, articles and so on.